VideoAPI/Video.API/Controllers/InstantMessageController.cs

```diff
@@ -53,12 +52,13 @@ public async Task<IActionResult> GetInstantMessageHistoryAsync(Guid conferenceId
             {
                 var messages =
                     await _queryHandler.Handle<GetInstantMessagesForConferenceQuery, List<InstantMessage>>(query);
+
                 var response = messages.Select(InstantMessageToResponseMapper.MapMessageToResponse);
                 return Ok(response);
             }
-            catch (ConferenceNotFoundException)
+            catch (Exception e)
```

Comment:
I think you should make this ConferenceNotFoundException again otherwise we are returning NotFound for ALL Exceptions, where we might want the call to fail on normal Exception

Reply:
We don't check for conference Id anymore, we get all IMs for conference, so no IMs could be found for existing or not existing conference. Exception can be thrown just in case of error on EF itself, so I don't know type of exception, I catch all of them.